### PR TITLE
adding back event_id and obs_id in dl1 tables

### PR DESCRIPTION
**Pull request by vuillaut**
`event_id` and `obs_id` disappeared from the dl1 image table with ctapipe v0.8

add simple test to check if these parameters are present in the file after r0 to dl1

### CODE REVIEW

File: lstchain/reco/r0_to_dl1.py [hunk 1]
```diff
@@ -659,10 +659,10 @@ def r0_to_dl1(
...
                     event.r0.prefix = ''
 
-                    writer.write(table_name = f'telescope/image/{tel_name}',
-                                 containers = [event.r0, tel, extra_im])
-                    writer.write(table_name = f'telescope/parameters/{tel_name}',
-                                 containers = [dl1_container])
+                    writer.write(table_name=f'telescope/image/{tel_name}',
+                                 containers=[event.index, tel, extra_im])
+                    writer.write(table_name=f'telescope/parameters/{tel_name}',
+                                 containers=[event.index, dl1_container])
```
Comment (maxnoe) [inline lstchain/reco/r0_to_dl1.py:665]: If you are now using the feature to write multiple containers at once, what is the point of having this large dl1 container?

Will event_id / obs_id be able to be read back in lstchain if it is still relying on everything being in that giant DL1 container?

Comment (vuillaut) [reply lstchain/reco/r0_to_dl1.py:665]: There is no good reason to use a big dl1 container anymore.
I just don't want to refactor everything if we move to stage1 soon (soon is a relative concept).

Comment (maxnoe) [reply lstchain/reco/r0_to_dl1.py:665]: The second point was the more important one...

Comment (vuillaut) [reply lstchain/reco/r0_to_dl1.py:665]: No, it won't.
It was not the case before, it still won't be.
Here I am merely trying to keep the level of features we already had.

Comment (vuillaut) [reply lstchain/reco/r0_to_dl1.py:665]: We can discuss starting the refactoring now, it's just not the purpose of this PR.

Comment (maxnoe) [reply lstchain/reco/r0_to_dl1.py:665]: Oh ok, I thought the problem was that it was there before (in the files) and it was expected to be read (e.g. in dl2) but if it wasn't read before, sure.

**Review by maxnoe — APPROVED** [review]

File: lstchain/reco/r0_to_dl1.py (hunk 1 above)
Comment (vuillaut) [reply lstchain/reco/r0_to_dl1.py:665]: Well, it is read in the flat table of parameters as we always do with pandas and alike.

Comment (maxnoe) [reply lstchain/reco/r0_to_dl1.py:665]: Ok, that was the question ;)

Comment (vuillaut) [reply lstchain/reco/r0_to_dl1.py:665]: Yes sorry there was a misunderstanding from me.
So it was in the table of parameters before and it is expected to be, e.g. to associate parameters to images (you could say the tables have a one to one correspondence but I prefer not to rely on this implicit correspondence).
However, it is not read back in containers as there is nothing doing that in lstchain at the moment.

**Pull request merged**